README.md

```diff
@@ -413,41 +413,45 @@ You can now make changes in the repository.
 
 ```bash
 gradlew test
 ```
 
-To shut down the Docker servers when you have finished testing:
+To run tests for the base driver functionality, use the following command:
+```bash
+gradlew test-community-docker
+```
 
+To run the integration tests, you will need some environment variables and an AWS Aurora database. Descriptions of the necessary environment variables is in the table below. To run the integration tests, use the following command (replace the <> tags with the appropriate values):
 ```bash
-$ cd aws-mysql-jdbc/docker
-$ docker-compose down && docker-compose rm
-$ cd ../
+DB_READONLY_CONN_STR_SUFFIX=<.cluster-ro-XYZ.us-east-2.rds.amazonaws.com> TEST_USERNAME=<username> TEST_PASSWORD=<password> TEST_DB_CLUSTER_IDENTIFIER=<db-identifier> DB_CONN_STR_SUFFIX=<.XYZ.us-east-2.rds.amazonaws.com> TEST_DB_USER=<jane_doe> gradlew test-integration-docker
 ```
+#### Environment Variables
+
+| Environment Variable          | Environment Variable Value                                                                                                                                                | Example Value                               |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------|
+| `DB_READONLY_CONN_STR_SUFFIX` | The URL suffix for the read-only cluster connection for your database cluster.                                                                                            | .cluster-ro-XYZ.us-east-2.rds.amazonaws.com |
+| `TEST_USERNAME`               | The name of the user to use while executing the tests.                                                                                                                    | username                                    |
+| `TEST_PASSWORD`               | The database cluster password.                                                                                                                                            | password                                    |
+| `TEST_DB_CLUSTER_IDENTIFIER`  | The database identifier for your Aurora cluster (found under the “DB Identifier” column when managing your RDS databases in AWS Management Console).                      | db-identifier                               |
+| `DB_CONN_STR_SUFFIX`          | The suffix URL pattern to use for connections that are made directly to an instance in your database cluster. This should also include the database to use for the tests. | .XYZ.us-east-2.rds.amazonaws.com            |
+| `TEST_DB_USER`                | User within the database that is identified with AWSAuthenticationPlugin. This is used for AWS IAM Auth.                                                                  | jane_doe                                    |
```

Comment:
AWSAuthenticationPlugin -> AWS IAM database authentication

This is used for AWS IAM Auth. -> This is used for AWS IAM authentication.